distribution/ddf-common/src/main/resources/bin/ddf_on_error.sh

```diff
@@ -15,7 +15,7 @@ waitForPid() {
    until kill -0 $1; do
      sleep 1
      i=$((i+1))
-     if [ $i -eq 60 ]; then
+     if [ $i -eq 120 ]; then
```

Comment:
❓ Any reason we changed from 1 to 2 minutes?

Reply:
Yeah! It was too short when starting from a service. It takes a bit more time to the service wrapper to detect the request to restart.

Reply:
Are we sure that 2 minutes is long enough then?

Reply:
yeah. From all the tests I have done, that was good enough. But even if it ain't we end up killing the process anyway forcibly. So either way we get what we want.